### DISCUSSION TITLE
Bootutil not compatible to Intel i210?

### DISCUSSION THREAD

**Discussion by sebastian859** [ipxe/ipxe#1217] [General]
Hi,
I was trying to install my build ROM on a Intel i210 inside an Advantech UNO-1372G-J021AE with the bootutil v1.2.72.0 linked on the regarding page https://ipxe.org/howto/romburning/intel
Unfortunately it seems this NIC is incompatible to the tool. Why I think so? According to "bootutil -help" it should display all supported Intel network ports found on the system. Instead it only returns the information about the tool and it's version. Same for the prompt "bootutil -e"

Can someone confirm this and/or maybe help? Thanks!

Br, Sebastian

**Comment by NiKiZe** [comment]
Onboard NICs seldom, if ever, has dedicated rom chips.

Since this is not an issue with iPXE, I moved this to discussions and closed the issue.
